privacy-dashboard/privacy-dashboard-impl/build.gradle

```diff
@@ -27,6 +28,7 @@ android {
 }
 
 dependencies {
+    implementation 'androidx.datastore:datastore-core-android:1.1.1'
```

Comment:
We don't keep versions of dependencies in module-level `build.gradle` files, but this line can be removed entirely since you've also added `api AndroidX.dataStore.preferences`